server/routes/api/documents/documents.ts

```diff
@@ -1278,15 +1272,7 @@ router.post(
     });
 
     ctx.body = {
-      data:
-        apiVersion === 2
```

Comment:
Technically a breaking API change but it should only have been used internally in previous versions